### PR TITLE
Devops ci/add license verification pipeline

### PR DESCRIPTION
**Pull request by nathan-bo**
added license verification pipeline and implement to ci

### CODE REVIEW

File: .github/workflows/license_verify.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: echo changed files
+        run: |
+          exit_code=0
+          for file in $(echo ${{needs.changedfiles.outputs.changed_files}})
+          do
+            if ! grep -q "Copyright (C) 2022 Speedb Ltd. All rights reserved." "$file"; then
```
Comment (rtpro) [inline .github/workflows/license_verify.yml:34]: @nathan-bo Please note that the year should not be part of the grep, as it would change next year.. :)

**Review by rtpro — CHANGES_REQUESTED** [review]

File: .github/workflows/license_verify.yml (hunk 1 above)
Comment (nathan-bo) [reply .github/workflows/license_verify.yml]: fixed, now it will use the current year instead of a set string :)

Comment (nathan-bo) [reply .github/workflows/license_verify.yml]: Changed it to regex to look for 20** as the year, let me know if you want this changed

Comment (maxb-io) [reply .github/workflows/license_verify.yml]: @nathan-bo I still see the hardcoded "2022". You can use regex for limiting to year 20XX: 
if ! grep -q "Copyright (C) 20.. Speedb Ltd. All rights reserved." "$file"; then
or catch the current year:
 grep -q  "Copyright (C) `date +%Y` Speedb Ltd. All rights reserved." 
 grep -q  "Copyright (C) $(date +%Y) Speedb Ltd. All rights reserved."

**Pull request closed without merging**